### ISSUE TITLE
Issue with population

### ISSUE THREAD

**Issue by wizardkang** [pmotschmann/Evolve#628]
Whenever I set my government to Democracy the game immediately starts to glitch out. Under the Civics tab I can see the numbers jumping back and forth between unemployed and one of the job categories until eventually I end up losing those workers. In my resource tab the numbers are doing the same thing, just going back and forth rapidly between categories like lumber and stone. The game stabilizes itself after a while but then it will randomly just start glitching again. My worker numbers might go back up sometimes but it never returns to the maximum like it is supposed to. I am not sure what to do, I did not have this issue yesterday though I chose the fungi path yesterday and today I with the plant path. I am not sure if that is the reason but I thought it might be worth mentioning.

**Comment by Beorseder** [comment]
Paste the save if it's still happening, but it just sounds like your people are starving to me.

**Comment by wizardkang** [comment]
Okay I ran into the issue again, I'm going to attach the save game file to this comment.
[evolve-2021-9-16.txt](https://github.com/pmotschmann/Evolve/files/7179984/evolve-2021-9-16.txt)

**Comment by Beorseder** [comment]
Yeah your people are starving. You need Food for population to grow, and each population consumes Food. You're never hitting max number of people because when new ones grow, they start consuming more Food which causes you to reach a negative amount of production that tanks Food stored down to 0 which prevents the growth of new population. If you have no Food and a negative enough production, then your people will start to starve, which is what's happening after the weather takes a turn for the worse and you're already slightly negative. You need to account for weather fluctuations either by having a significant amount more Farmers than you would need during normal conditions and/or by having more Food storage to survive longer during periods of bad weather.
